libcalico-go/lib/health/health.go

```diff
@@ -144,27 +147,29 @@ func NewHealthAggregator() *HealthAggregator {
 		reporters:    map[string]*reporterState{},
 		httpServeMux: http.NewServeMux(),
 	}
-	aggregator.httpServeMux.HandleFunc("/readiness", func(rsp http.ResponseWriter, req *http.Request) {
-		log.Debug("GET /readiness")
+	genResponse := func(rsp http.ResponseWriter, quality string, state bool, detail string) {
```

Comment:
any reason we need this to be an inline function? Might be easier to read if we make it a proper function.

Reply:
Good idea, done.